code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1600,105 +1598,6 @@ func (srh *StorageRestHandler) getLatestReadMarker(w http.ResponseWriter, r *htt
 	}
 }
 
-// swagger:route GET /v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/allocation_min_lock allocation_min_lock
-// Calculates the cost of a new allocation request.
-//
-// parameters:
-//
-//	+name: allocation_data
-//	 description: json marshall of new allocation request input data
-//	 in: query
-//	 type: string
-//	 required: true
-//
-// responses:
-//
-//	200: Int64Map
-//	400:
-//	500:
-func (srh *StorageRestHandler) getAllocationMinLock(w http.ResponseWriter, r *http.Request) {
```

Comment:
return allocation cost here, since UI relies on this api and removing it we can breakweb-apps

Reply:
UI is not using this API at all. I don't see use case of this API and this is not used even in gosdk.